chart/flux/templates/deployment.yaml

```
@@ -45,6 +45,12 @@ spec:
       - name: git-keygen
         emptyDir:
           medium: Memory
+      {{- if .Values.registry.acr.enabled }}
+      - name: acr-credentials
+        hostPath:
+          path: "{{ .Values.registry.acr.mountPath }}"
```

```suggestion
          path: "{{ .Values.registry.acr.hostPath }}"
```

as per docs and `values.yaml` example.

---

Thanks for the catch! Looks like this is the only place I made that mistake.